src/Infrastructure/Controller/MyArea/Organization/DetailOrganizationController.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Infrastructure\Controller\MyArea\Organization;
+
+use App\Application\QueryBusInterface;
+use Symfony\Bundle\SecurityBundle\Security;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Routing\Requirement\Requirement;
+
+final class DetailOrganizationController extends AbstractOrganizationController
```

Comment:
(Détail) Naming en cohérence avec le `RegulationDetailController` et `app_config_organization_detail` ?

```suggestion
final class OrganizationDetailController extends AbstractOrganizationController
```

Reply:
@mmarchois Tu as marqué comme résolu mais ça n'a pas été changé (pour info)